src/dmd/identifier.d

```diff
@@ -141,6 +141,14 @@ nothrow:
         return DYNCAST.identifier;
     }
 
+    /// Returns `true` if this is not anonymous and starts with `prefix`
+    extern(D) final bool startsWith(const scope string prefix) const pure @nogc @safe
```

Comment:
Change parameter to `const scope char[] prefix` to allow non-immutable array prefixes.